### PR TITLE
Geowave 999

### PR DESCRIPTION
**Pull request by mak438**
#999

### CODE REVIEW

**Comment by mak438** [comment]
Apparently one of the new tests segfaulted during the CI build. Looking into that.

File: extensions/cli/landsat8/src/main/java/mil/nga/giat/geowave/format/landsat8/Landsat8IngestCommand.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 {
 
 	@Parameter(description = "<storename> <comma delimited index/group list>")
-	private final List<String> parameters = new ArrayList<String>();
+	protected final List<String> parameters = new ArrayList<String>();
```
Comment (mak438) [inline extensions/cli/landsat8/src/main/java/mil/nga/giat/geowave/format/landsat8/Landsat8IngestCommand.java]: Note to self: undo this.

File: extensions/cli/landsat8/src/main/java/mil/nga/giat/geowave/format/landsat8/AnalyzeRunner.java [hunk 2]
```diff
@@ -181,6 +181,10 @@ private void printSceneInfo() {
 					final String bandId = entry.getKey();
 					final SimpleFeature feature = entry.getValue();
 					if (first) {
+						if (System.out == null) throw new RuntimeException(
```
Comment (mak438) [inline extensions/cli/landsat8/src/main/java/mil/nga/giat/geowave/format/landsat8/AnalyzeRunner.java]: Note to self: undo this.

**Comment by rfecher** [comment]
resubmitted as PR #1036

I tweaked the filter to add ".. and sizeMB < 1" just to make sure we're getting small files (considering there was no time constraint, as imagery is collected in the future there are no guarantees of the size without this.

I also add calls to set the scale which essentially scales the image first before ingest, and it therefore finished quickly.

Lastly, just squashed commit history down

**Pull request closed without merging**